src/api/document.js

```diff
@@ -140,6 +140,7 @@ export async function deleteDocument(ids) {
   addDeletedDocs(ids);
 }
 
+// "metadata" here includes access level, publish date, and noindex.
```

Comment:
I believe the metadata here can be any attribute on document - not sure exactly where its called, but the comment above and below seem to be slightly misleading